src/Pages/kiwanlee/Login/Login.scss

```
@@ -0,0 +1,88 @@
+* {
+  box-sizing: border-box;
+
+  body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+    .Login {
+      height: 100vh;
+
+      .container {
+        display: flex;
+        justify-self: center;
+        align-items: center;
+        height: 100%;
+        .inner {
+          width: 350px;
+          height: 380px;
+          padding: 10px 40px;
+          box-sizing: border-box;
+          background: #fff;
+          border: 1px solid #dbdbdb;
+          .header {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin: 30px 0;
+            h1 {
+              display: flex;
+              font-family: 'Lobster', cursive;
+            }
+          }
```

```suggestion
      .container {
        display: flex;
        justify-self: center;
        align-items: center;
        height: 100%;
        
        .inner {
          width: 350px;
          height: 380px;
          padding: 10px 40px;
          box-sizing: border-box;
          background: #fff;
          border: 1px solid #dbdbdb;
          
          .header {
            display: flex;
            justify-content: center;
            align-items: center;
            margin: 30px 0;
            
            h1 {
              display: flex;
              font-family: 'Lobster', cursive;
            }
          }
```
이처럼 가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.

---

해당 부분 뿐만 아니라 전체적으로 수정해주세요!
또한 
- 하나의 요소에 여러가지 속성을 부여하는 경우 중요도, 관련도에 따라서 나름의 convention을 지켜서 작성하는 것이 좋습니다.
- 일반적인 convention 은 아래와 같습니다. 아래와 같이 순서 적용해주세요.

    [CSS property 순서]

    - Layout Properties (position, float, clear, display)
    - Box Model Properties (width, height, margin, padding)
    - Visual Properties (color, background, border, box-shadow)
    - Typography Properties (font-size, font-family, text-align, text-transform)
    - Misc Properties (cursor, overflow, z-index)